schema_based_on_diagram.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE patients(
+   id serial Primary key,
+   name text,
+   date_of_birth date
+);
+
+CREATE TABLE treatments(
+   id serial Primary key,
+   type text,
+   name text
+);
+
+CREATE TABLE medical_histories(
+   id serial Primary key,
+   admitted_at timestamp,
+   patient_id integer References patients (id),
+	status text
+);
```

Comment:
- [ ] Please 🙏🏽 add the FK indexes. For example: `CREATE INDEX ON medical_histories (patient_id);`

This is a requirement of the project 👇🏽 

> Remember to add the FK indexes.